website/docs/d/eventhub_namespace.html.markdown

```diff
@@ -44,6 +44,8 @@ output "eventhub_namespace_id" {
 
 * `zone_redundant` - Is this EventHub Namespace deployed across Availability Zones?
 
+~> **Note:** The `zone_redundant` field will be removed in v4.0 of the provider.
```

Comment:
Anyone using the field will be notified by the deprecation messages added to the field so no need to add this to the documentation as well
```suggestion
```

Reply:
Fixed.